src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int INCREASE_THE_LENGTH = 2;
+    private Node<K,V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
it's better to initialize this field in the constructor

Reply:
Also, you need a variable for currentLoad, instead of counting it during resize
currentLoad = (int) (DEFAULT_SIZE * DEFAULT_LOAD_FACTOR)